GeneralUtilities/src/SConscript

```diff
@@ -8,6 +8,7 @@
 import os
 Import('env')
 Import('mu2e_helper')
+rootlibs = env['ROOTLIBS']
```

Comment:
Do you need all of ROOTLIBS or will a much shorter list work?

Reply:
it needs ROOT::Math.  I can try that...